modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,6 +14,12 @@ Provide the following info for each issue if possible:
 *Result* - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
+The following issues have been resolved in the current release:
+
+* https://issues.redhat.com/browse/OSSM-4851[OSSM-4851] Previously, an error occurred in the operator deploying new pods in a namespace scoped inside the mesh when `runAsGroup`, `runAsUser`, or `fsGroup` were `nil`. Now, a yaml validation has been added to avoid the `nil` value.
```

Comment:
```suggestion
* https://issues.redhat.com/browse/OSSM-4851[OSSM-4851] Previously, an error occurred in the operator deploying new pods in a namespace scoped inside the mesh when `runAsGroup`, `runAsUser`, or `fsGroup` specifications were `nil`. Now, a yaml validation has been added to avoid the `nil` value.
```

Reply:
I'm not really sure what the appropriate noun is for those.

Reply:
That is a good point! I'm sure if it's "specifications" or "variables" or something else so I will check. Thanks for catching that!

Reply:
Parameters. They are parameters. Updated accordingly.